src/styles.scss

```diff
@@ -1,2 +1,3 @@
 /* You can add global styles to this file, and also import other style files */
 @import "~nhsuk-frontend/packages/nhsuk";
+@import "~@angular/material/prebuilt-themes/indigo-pink.css";
```

Comment:
unsure about importing the whole sass library as such. i.e. compiled css pink theme. will have to customize it using nhs colours which i previously did.

Reply:
yea this makes sense - i'll remove the prebuilt theme and have a look at building a custom one to leverage nhs variables ui colours etc